bower.json

```diff
@@ -39,7 +39,7 @@
     "angular-sanitize": "1.3.0 - 1.5.*",
     "angular-bootstrap": "0.13.x",
     "lodash": "3.x",
-    "patternfly": "~3.7.0"
+    "patternfly": "git://github.com/patternfly/patternfly.git#master-dist"
```

Comment:
I question the need to do this. If Angular Patternfly has a dependency on bootstrap-select (and specifically now on 1.10) then shouldn't this dependency be here with the necessary version?